README.md

```diff
@@ -4,6 +4,7 @@
 <a href="https://in.waw.pl/systemd-github-state/systemd-systemd-pull-requests.svg"><img align="right" src="https://in.waw.pl/systemd-github-state/systemd-systemd-pull-requests-small.svg" alt="Count of open pull requests over time"></a>
 [![Semaphore CI Build Status](https://semaphoreci.com/api/v1/projects/28a5a3ca-3c56-4078-8b5e-7ed6ef912e14/443470/shields_badge.svg)](https://semaphoreci.com/systemd/systemd)<br/>
 [![Coverity Scan Status](https://scan.coverity.com/projects/350/badge.svg)](https://scan.coverity.com/projects/350)<br/>
+[![Fuzzit Status](https://app.fuzzit.dev/badge?org_id=RxqRpGNXquIvqrmp4iJS&target_id=2ODbhEjfRF2AZtrUotMh&branch=master)](http://localhost:3000/admin/RxqRpGNXquIvqrmp4iJS/dashboard)<br/>
```

Comment:
Looks like the url points to `localhost` (which I assume was used for debugging). Could you change it to `app.fuzzit.dev`?

Just to be sure, does this badge show the status of `fuzz-bus-label` only? Is there a way to show the global status, which I think should be "green" when all the fuzz targets are green?

Reply:
Yes, will fix now for both questions.